### PR TITLE
refactor(center-story): stories now centered with official package #479

### LINKED ISSUE #479: [Storybook] Refactor how stories are centered
Storybook has an [official addon for centering stories.](https://github.com/storybookjs/storybook/tree/next/addons/centered)

We should move to this and deprecate our wrapper that does this. Should help us retain the ability for stories to be full width and still center, without having custom css.

### PR DESCRIPTION
**Pull request by sstone2423**
Closes #479

**Summary**

- @storybook/addon-centered --dev package added as the new standard for
centering stories. https://github.com/storybookjs/storybook/tree/next/addons/centered

- imported the new package and added the decorator in config.js

- Removed in-line styling from .storybook/Container.js as the new package handles it

- Updated the StoryShot snapshots because the new styling added new lines which shifted the rest of the stories down

**Change List (commits, features, bugs, etc)**

- refactor(center-story): stories now centered with official package #479

**Acceptance Test (how to verify the PR)**

- Check the built Storybook to see that each story is indeed centered. https://deploy-preview-790--carbon-addons-iot-react.netlify.com/

### CODE REVIEW

File: .storybook/Container.js [hunk 1]
```diff
@@ -5,18 +5,6 @@ export default class Container extends Component {
   render() {
     const { story } = this.props;
 
-    return (
-      <div
-        className="storybook-container"
-        style={{
-          padding: '3rem',
```
Comment (tay1orjones) [inline .storybook/Container.js]: The stories that are larger than the viewport height are put right on the top/bottom edge.  Left this PR, right production:

image

Would like to keep the breathing room via the padding if possible.

Comment (sstone2423) [reply .storybook/Container.js]: Re-added the padding. Thanks!

**Comment by stuckless** [comment]
One of the challenges that I see with this 'centering' approach is that it is very hard to tell what padding/margin the component is exporting by looking at it.  Some components have padding, some don't.  This makes it hard to tell.

**Comment by sstone2423** [comment]
@stuckless While I understand your point, Carbon also centers their components in stories. It seems right to continue their style

**Comment by davidicus** [comment]
@sstone2423 we just need to resolve this conflict and we can merge this.

**Pull request merged**